packages/react/src/VideoPlayer/components/Captions/Captions.tsx

```diff
@@ -39,13 +39,13 @@ export const Captions = ({className, ...props}: HTMLAttributes<HTMLDivElement>)
     return () => videoRef.removeEventListener('timeupdate', compareAndSetCaption)
   }, [videoRef, trackInformation])
 
-  if (!caption) {
+  if (!trackInformation) {
```

Comment:
Ensure the captions bar is always visible if captions are enabled and available. This prevents the height of the VideoPlayer reducing when no captions are visible.